### ISSUE TITLE
OrderBy with ColumnAlias issue

### ISSUE THREAD

**Issue by iXnaL** [Raizlabs/DBFlow#327]
I have a master table and a child table both have a column named __`id`__, and the child table has a column named _eg._ __`master_id`__ related to the master table.

When I write a query like below

```javascript
new Select()
                .from(MasterTable.class).as("mt")
                .join(ChildTable.class, Join.JoinType.INNER).as("ct")
                .on(Condition.column(ColumnAlias.columnWithTable("mt", MasterTable_Table.ID)).is(ColumnAlias.columnWithTable("ct", ChildTable_Table.MASTER_MT_ID)))
                .where(queryBuilder)
                .orderBy(OrderBy.columns(ColumnAlias.columnWithTable("mt", MasterTable_Table.ID).shouldQuoteColumnName(false)).descending())
                .limit(pageSize);
```

The query output the SQL like below

```javascript
SELECT * 
FROM `MasterTable` AS mt 
INNER JOIN `ChildTable` AS ct ON `mt`.`id`=`ct`.`master_id` 
WHERE ...  
ORDER BY ``mt``.``id`` DESC 
LIMIT 20 
```

I have tried adding or removing the __shouldQuoteColumnName(false)__ for ColumnAlias, but it seems that this method would not impact the result.

**Comment by agrosner** [comment]
@iXnaL can you explain to me what is wrong in this case?

**Comment by iXnaL** [comment]
@agrosner I got an error when I try to use the select method orginally, I trace into the source code and find that the order by column has double backticks, I thought it's a syntax error at that time because it's illegal in mysql. Although finally I realized maybe SQLite had ignored this syntax, I still think double backticks are redundant.

**Comment by meneerraaf** [comment]
If I'm not mistaken, `Where.java` should have

```
public Where<ModelClass> orderBy(boolean ascending, ColumnAlias... columns)
```

Because `OrderBy.java` already supports

```
public static OrderBy columns(ColumnAlias... columnAliases)
```

You can then use:

```
orderBy(true, ColumnAlias.columnWithTable("A", Some$Table.COLUMN))
```

**Comment by agrosner** [comment]
fixed in 3.0/develop
